Model/Ofeq.js

```diff
@@ -1,8 +1,28 @@
-import Satelite from "./Satelite";
-
 class Ofeq extends Satelite {
-  constructor(model, revolution, fuel, generation, favStar, wings) {
-    super("EO", "Ofeq", model, 350, revolution, fuel);
+  constructor(
+    screenX,
+    screenY,
+    height,
+    width,
+    model,
+    revolution,
+    fuel,
+    generation,
+    favStar,
```

Comment:
You sort of mixed the satellite here with the previous problem - no reason I think for a favStar, but whatever.